doc/user/content/sql/namespaces.md

```diff
@@ -9,16 +9,73 @@ menu:
     weight: 126
 ---
 
-Materialize mimics SQL standard's namespace hierarchy, which is:
+Namespaces are a logical way to organize Materialize objects. In
+organizations with multiple objects, they help avoid naming conflicts and make
+it easier to manage objects. Namespaces allow you to refer to objects by a fully
+qualified name like `namespace.object_name`.
+
+## Namespace hierarchy
+
+Materialize follows SQL standard's namespace hierarchy for most objects. The
+Materialize structure is:
 
 - Databases (highest level)
 - Schemas
-- Tables, views, sources
+    - Tables
+    - Views
+    - Materialized views
+    - Sources
+    - Sinks
+    - Indexes
+    - Types
+    - Functions
+    - Secrets
```

Comment:
What about connections?

Reply:
Added it above sources. Based in the `mz_catalog.mz_connections` table, they are linked to a `schema_id`